tensorflow/g3doc/tutorials/seq2seq/index.md

```diff
@@ -25,11 +25,17 @@ prepare it for training and train. It takes about 20GB of disk space,
 and a while to download and prepare (see [later](#lets-run-it) for details),
 so you can start and leave it running while reading this tutorial.
 
+This tutorial references the following files from `python/ops/`.
+
+File | What's in it?
+--- | ---
+`python/ops/seq2seq.py` | Library for building sequence-to-sequence models.
```

Comment:
Mind merging this into one list? Something like:

This tutorial references the following files:
python/ops/seq2seq.py ...
models/rnn/translate/...